hathor/consensus/consensus_settings.py

```diff
@@ -63,19 +63,36 @@ def _get_valid_vertex_versions(self, include_genesis: bool) -> set[TxVersion]:
         }
 
 
-class PoaSettings(_BaseConsensusSettings):
-    type: Literal[ConsensusType.PROOF_OF_AUTHORITY] = ConsensusType.PROOF_OF_AUTHORITY
-
-    # A list of Proof-of-Authority signer public keys that have permission to produce blocks.
-    signers: tuple[bytes, ...]
+class PoaSignerSettings(BaseModel):
+    public_key: bytes
+    start_height: NonNegativeInt | None = None
```

Comment:
You can remove the `None` choice and simply set `start_height = 0` by default.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1075/commits/ad736ff38c19f9acc9f76531cb223de85d6eb724

Reply:
Thanks!